host/xtest/xml/include/xml_timearithm_api.h

```diff
@@ -366,45 +367,15 @@ static TEEC_UUID UUID_TTA_testingInternalAPI_Time = {
 };
 
 /*Helper functions/macros*/
-#define IDENTIFIER_NOT_USED(x) { if (sizeof(&x)) {} }
-
-#define ALLOCATE_SHARED_MEMORY(context, sharedMemory, sharedMemorySize, \
-			       memoryType) \
-	res = AllocateSharedMemory(context, sharedMemory, sharedMemorySize, \
-				   memoryType); \
-	if (res != TEEC_SUCCESS) { \
-		goto exit; \
-	} \
-	memset(sharedMemory->buffer, 0, sharedMemorySize);
-
-#define ALLOCATE_AND_FILL_SHARED_MEMORY(context, sharedMemory, \
```

Comment:
This one is different, is has 5 parameters. The other versions you have removed (in other .h files) had 6 parameters.
As a result the macro in `xml_common_api.h` does not work in `xml_timearithm_api.h`, and then `xtest_8500.c` fails to compile (it is a GP test).